### ISSUE TITLE
Warning when changing both of bounds

### ISSUE THREAD

**Issue by thiagonovaesb** [jump-dev/Xpress.jl#175]
```
function MOI.set(
    model::Optimizer,
    ::MOI.ConstraintSet,
    c::MOI.ConstraintIndex{MOI.VariableIndex, S}, s::S
) where {S<:SCALAR_SETS}
    MOI.throw_if_not_valid(model, c)
    lower, upper = _bounds(s)
    info = _info(model, c)
    if lower !== nothing
        _set_variable_lower_bound(model, info, lower)
    end
    if upper !== nothing
        _set_variable_upper_bound(model, info, upper)
    end
    info.previous_lower_bound = _get_variable_lower_bound(model, info)
    info.previous_upper_bound = _get_variable_upper_bound(model, info)
    return
end
```
Once this function call `_set_variable_lower_bound` before `_set_variable_upper_bound`, it can produce a warning about Inconsistent bounds

**Comment by odow** [comment]
So you have a reproducible example of the problem?

**Comment by thiagonovaesb** [comment]
@odow here is an example, this input:
```
using MathOptInterface
using Xpress
# use your xpress license
model = Xpress.Optimizer()
x = MOI.add_variable(model)
c = MOI.add_constraint(model, x, MOI.EqualTo(2.0))
MOI.set(model, MOI.ConstraintSet(), c, MOI.EqualTo(3.0))
```
Return this warning:
`?475 Warning: Inconsistent bounds [3,2] for column C1 in call to XPRSchgbounds`

**Comment by joaquimg** [comment]
The issues is:
We call:
```julia
Xpress.chgbounds(model.inner, [info.column], Cchar['L'], [value])
```
Then:
```julia
Xpress.chgbounds(model.inner, [info.column], Cchar['U'], [value])
```
So for an instant bounds are inconsistent. but after the second call everything is ok.
We could call:
```julia
Xpress.chgbounds(model.inner, [info.column], Cchar['B'], [value])
```
if we are fixing variables, that would change both at the same time and avoid the warning.

**Comment by joaquimg** [comment]
fixed by #187
